src/core/sock/sockinfo_udp.cpp

```diff
@@ -1127,31 +1127,16 @@ int sockinfo_udp::setsockopt(int __level, int __optname, __const void *__optval,
                 break;
             }
 
-            // Find local interface IP address
+            // Find local if for this MC ADD/DROP
```

Comment:
[suggestion] `local interface` instead of `local if`.

Reply:
Done